### DISCUSSION TITLE
How to use isar with Native Kotlin or swift?

### DISCUSSION THREAD

**Discussion by lijianqiang12** [isar/isar#960] [Help]
How can i use isar with Native Kotlin or swift? I want to use isar in flutter and Android/iOS Widget, but i donot know how to do it.

**Comment by armandsLa** [comment]
I'm also wondering if this is possible. I couldn't find anything in the docs.

This is possible with Sqflite and Realm database.

**Reply by galacticgibbon** [reply]
I suppose it's also possible with the drift database, since that uses sqlite under the hood
